GsaGH/Components/0_Model/CreateModel.cs

```diff
@@ -1,8 +1,10 @@
-﻿using System;
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
```

Comment:
Do we need need this license for each file?

Reply:
deleted